src/event_listener.cc

```diff
@@ -0,0 +1,21 @@
+#include <fmt/chrono.h>
+#include <fmt/core.h>
+#include <fmt/ranges.h>
+
```

Comment:
```suggestion
```

These seem to not be used here.

Reply:
They're a relic of the past. I'll remove them!